sdk/go/common/util/gitutil/git.go

```diff
@@ -650,6 +650,14 @@ type gitRepoURLParts struct {
 }
 
 func parseGitRepoURLParts(rawurl string) (gitRepoURLParts, error) {
+	urlRegex := regexp.MustCompile(`^[^\./][a-zA-Z0-9-]*\.[a-z]+/[a-zA-Z0-9-/]+$`)
+	if urlRegex.MatchString(rawurl) {
```

Comment:
Can we not just check that there _isn't_ a scheme at the start of the url already? That's probably a simpler regex.

Reply:
Yeah that's probably good enough actually.  I was worried about still allowing local repos working, but thinking again, we actually never allowed that, so there's no point in worrying about those (and I don't think most people would want that anyway)